{{cookiecutter.project_slug}}/README.rst

```diff
@@ -6,6 +6,52 @@ Introduction
 Usage, etc.
 
 
+Development
+-----------
+
+This project makes use of `Pipenv <https://docs.pipenv.org/>`_., which creates
+a project virtual environment. We like to keep this virtual environment inside
+``{{ cookiecutter.project_slug }}/.venv``. This is not the default Pipenv
+behaviour, so we need to set the following environment variable:
+``export PIPENV_VENV_IN_PROJECT=1``. If you add that to your ``.bashrc``, you
```

Comment:
Ik vermoed dat je het ook *lokaal* ergens in kan stellen. In de pipfile of pyproject.toml ofzo. Ik weet het niet zeker.

Reply:
Niet dat ik weet, en ik heb er even naar gezocht.